### ISSUE TITLE
bower_components .css

### ISSUE THREAD

**Issue by Grandiz** [Swiip/generator-gulp-angular#410]
Hi everyone.
I'm just faced to some issue. When i tried gulp serve, everything worked perfectly, but when i run gulp serve:dist, some css files from bower still injected to html.
Could you give the hand how to fix it

![image](https://cloud.githubusercontent.com/assets/9084338/6543093/d325dfe2-c516-11e4-9a63-b4de7e45b4dc.png)

**Comment by Swiip** [comment]
How have you inserted your css files in the `index.html`, have you let wiredep inject it automatically?

**Comment by Grandiz** [comment]
thanks for reply. i didn't change index.html.
    
    
    
    

    
    
    
    
    


and inject.js
```javascript
'use strict';

var gulp = require('gulp');

var paths = gulp.paths;

var $ = require('gulp-load-plugins')();

var wiredep = require('wiredep').stream;

gulp.task('inject', ['styles'], function () {

  var injectStyles = gulp.src([
    paths.tmp + '/serve/{app,components}/**/*.css'
  ], { read: false });

  var injectScripts = gulp.src([
    paths.src + '/{app,components}/**/*.js',
    '!' + paths.src + '/{app,components}/**/*.spec.js',
    '!' + paths.src + '/{app,components}/**/*.mock.js'
  ]).pipe($.angularFilesort());

  var injectOptions = {
    ignorePath: [paths.src, paths.tmp + '/serve'],
    addRootSlash: false
  };

  var wiredepOptions = {
    directory: 'bower_components',
    exclude: [/bootstrap\.css/, /foundation\.css/]
  };

  return gulp.src(paths.src + '/*.html')
    .pipe($.plumber())
    .pipe($.inject(injectStyles, injectOptions))
    .pipe($.inject(injectScripts, injectOptions))
    .pipe(wiredep(wiredepOptions))
    .pipe(gulp.dest(paths.tmp + '/serve'));

});

```

**Comment by Grandiz** [comment]
in my opinion it doesn't build .css from bower_components at all

**Comment by bluesliverx** [comment]
I've run into this same problem.

**Comment by AlexanderTserkovniy** [comment]
+1

**Comment by zckrs** [comment]
Fixed on master branch

**Comment by AlexanderTserkovniy** [comment]
How to get this fix in my current version? Is there possibility to update?

**Comment by zckrs** [comment]
Update a generator is not safe.
Wait next release and create a new project with this release

**Comment by AlexanderTserkovniy** [comment]
But if my project is already quite big, is it possible to just add this functionality only?

**Comment by zckrs** [comment]
The next version while include Toastr.
You can try but you should manually check change.

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada:
Your issue should be fixed. I close but if you can retry your case with a new fresh install of generator you are welcome. Any feedback would be appreciated (positive comments too ^_^).
